ledger/store/trackerdb/sqlitedriver/schema.go

```diff
@@ -948,11 +948,23 @@ func accountsAddCreatableTypeColumn(ctx context.Context, e db.Executable, popula
 	var creatableTypeOnResourcesRun bool
 	err := e.QueryRow("SELECT 1 FROM pragma_table_info('resources') WHERE name='ctype'").Scan(&creatableTypeOnResourcesRun)
 	if err == nil {
-		// Already exists.
+		// Check if any ctypes are invalid
+		var count uint64
+		err0 := e.QueryRow("SELECT COUNT(*) FROM resources WHERE ctype NOT IN (0, 1)").Scan(&count)
+		if err0 != nil {
+			return err0
+		}
+		if count > 0 {
+			// Invalid ctypes found, return an error
+			return fmt.Errorf("invalid ctypes found in resources table; database is corrupted and needs to be rebuilt")
```

Comment:
add repair steps?
1. ensure the DB version is below 11 by running `PRAGMA user_version;`
2. drop ctype column `ALTER TABLE resources DROP COLUMN ctype;`
3. restart algod

Reply:
I don't think these steps are safe over time - what if we add migration 12 next that does something completely different? Setting the db version back could cause less than ideal behavior in that scenario.